packages/varlock-website/src/content/docs/plugins/1password.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: 1Password Plugin
+description: Using 1Password with Varlock
+---
+
+import { Steps, Icon } from '@astrojs/starlight/components';
+
+Our [1Password](https://1password.com/) plugin enables secure loading of values from 1Password vaults using declarative instructions within your `.env` files.
+
+For local development, it (optionally) supports authenticating using the local 1Password desktop app, including using biometric unlock. Otherwise it uses a [service account](https://developer.1password.com/docs/service-accounts/) making it suitable for CI/CD and production environments.
```

Comment:
```suggestion
For local development, it (optionally) supports authenticating using the local 1Password desktop app, including using biometric unlock. Otherwise, it uses a [service account](https://developer.1password.com/docs/service-accounts/) making it suitable for CI/CD and production environments.
```